packages/util/src/time.ts

```diff
@@ -93,6 +95,8 @@ export function getDateString(
     return `${date.toLocaleDateString('en-US')}`;
   } else if (opts?.hideYear) {
     return `${month} ${day}`;
+  } else if(opts?.monthDayDigits) {
```

Comment:
Can you remove this? I don't wan't to add more date formats. Just use the default here.